### ISSUE TITLE
Error in func 'Etash_cuda_miner::search'

### ISSUE THREAD

**Issue by dabingus** [ethereum-mining/ethminer#80]
error in func 'ethash_cuda_miner::search' at line 365 : unspecified launch failure.

all 6 of my 1060s go to 0 hash and this happens after about on 30 minutes of mining.
I am using 55 power limit,66 temp limit, 0 core clock, +700 memory. my gpus do not get hotter than 66C. These setting run great in claymore with no isssues. Hopefully this error can be resolved.
![error-etherminer](https://user-images.githubusercontent.com/1150896/27639720-b7057d72-5bdc-11e7-87de-c1e7ad0c42ef.PNG)

**Comment by shanemgrey** [comment]
I'm getting **almost** the exact same error.  Seems to not matter what my overclock is at.  

Using 2 PNY GTX 1060 cards in an ASUS z270a motherboard with the newest nvidia 381.22 driver on Ubuntu 16.04.2, running with lightdm started.

Happens at random, sometimes a couple minutes after start, sometimes an hour after.  

Exact error for mine is 

    CUDA error in func 'search' at line 365 : an illegal instruction was encountered
    .
    X 17:10:19 | cudaminer 1  Error CUDA mining: an illegal instruction was encountered

Then repeat for cudaminer 0

I'm using the most recent 0.11.0rc1 of ethminer with the nvidia specific updates.  Haven't used prior versions enough to know if the error was there as well.

**Comment by shanemgrey** [comment]
Likely duplicate of #72

**Comment by rizwansarwar** [comment]
Yes exactly the same problem as I reported in #72 . Some more information, I have these running on two separate systems (exactly same release rc11). One is running on a system with Intel i5 CPU and the other one is running on an AMD FX4300 CPU. So far I have only observed crashed on AMD system. I have tried to swap cards in systems too to find out if it was card specific. 

I am going to run the AMD system on slightly lower clock to see if it is the overclocking causing the problem. Will report back.

**Comment by bcamur** [comment]
I started to get this error after updating to 0.11.0rc1. Unlike others having the same problem I think I can reproduce the error consistently. Most of the time my miner runs fine for hours uninterrupted but whenever I try to wake up the screen after a long period of mining (like after waking up in the morning so say 7-8 hours of continuous mining) by moving the mouse etc. it hangs for 15-30 seconds and then wakes the screen up with this error. This does not happen when I try to wake the screen up after short periods of mining (say 10-15 minutes) and miner has even run for 2 days uninterrupted in some cases when I never tried to use the screen. Hope this helps.

**Comment by derubm** [comment]
m  15:09:25|main  Mining on PoWhash #82d0ae56 : 25.21MH/s [A3+0:R0+0:F0]
  m  15:09:26|main  Mining on PoWhash #82d0ae56 : 25.21MH/s [A3+0:R0+0:F0]
CUDA error in func 'ethash_cuda_miner::search' at line 365 : an illegal memory access was encountered.
  X  15:09:26|cudaminer0  Error CUDA mining: an illegal memory access was encountered
  m  15:09:26|main  Mining on PoWhash #82d0ae56 : 0.00MH/s [A3+0:R0+0:F0]
  m  15:09:27|main  Mining on PoWhash #82d0ae56 : 0.00MH/s [A3+0:R0+0:F0]
same here, interesting thing for me is : it happens with all versions except the very first dev version ?

**Comment by oleng** [comment]
funny, it seems from the comments, this error & [https://github.com/ethereum-mining/ethminer/issues/72](url) only (or mostly) hits 1060. Mine are 3GB Zotac AMP with Hynix memory, higher OC would means faster error & crashes. But this happened with genoil too, so i thought at first it was a case of bad memory in my cards. Didn't check with hwinfo though.

Happens most often in windows 10, but also in ubuntu, binary release & built from source.

**Comment by chfast** [comment]
Have you tried running with `--cuda-parallel-hash 1`?
If this is a duplicate of #72, can we move the discussion there?

**Comment by DeadManWalkingTO** [comment]
Any news?
